src/app/checkout/step-3/step-3.component.js

```diff
@@ -81,6 +81,7 @@ class Step3Controller{
   }
 
   submitOrder(){
+    this.submissionError = '';
```

Comment:
It's essentially pick your favorite version of false... I can `delete` it. I was just trying to remove the error from the top once a new request is sent.